Take 2.1/Assets/Scripts/Controls.cs

```diff
@@ -37,13 +43,14 @@ void Start()
     void Update()
     {
         //grabs horizontal movement input
-        inputX = Input.GetAxis("Horizontal");
+        inputX = Input.GetAxisRaw("Horizontal");
+        inputY = Input.GetAxisRaw("Vertical");
 
-        if (Input.GetKeyDown("space"))
+        if (Input.GetKeyDown("f"))
```

Comment:
I thought we decided on changing the attack key to enter?

Reply:
I forgot about this ^

Reply:
ok now worries, I can change that